test/extended/images/mysql_replica.go

```diff
@@ -27,8 +27,8 @@ var (
 		{
 			"5.5",
 			"https://raw.githubusercontent.com/openshift/mysql/master/5.5/examples/replica/mysql_replica.json",
-			// TODO: Investigate why this is broken.
-			true,
+			// NOTE: Set to true in case of flakes.
```

Comment:
This was either not working or causing flakes in the past. While testing 5.6, I haven't seen 5.5 fail, so I decided to enable it again. However, I'm worried that it is possible flakes will appear, that's why I added the comment.